hspec-core/.gitignore

```diff
@@ -0,0 +1 @@
+.hspec/id/
```

Comment:
Hspec reads options from `.hspec` and `~/.hspec`.   So this directory name may conflict with this.